.github/workflows/update-pr-labels.yml

```
@@ -0,0 +1,29 @@
+name: Update PR labels
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  after-merge:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+
+    steps:
+      - uses: cachix/install-nix-action@v30
+
+      - run: nix-env -iA nixpkgs.github-cli
+
+      - name: Update labels on accepted RFCs
+        env:
+          GH_TOKEN: ${{ github.token }}
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+          REPO: ${{ github.repository }}
+        run: |
+          # Only operate on PR if it has the `status: accepted` label
+          if gh pr view $PR_NUMBER --repo $REPO --json labels -q 'any(.[].name == "status: accepted")'; then
+            gh pr edit $PR_NUMBER --remove-label "status: accepted" --repo $REPO
+            gh pr edit $PR_NUMBER --add-label "status: merged" --repo $REPO
+          fi
```

We don't have a "merged" label, but I think the goal was to add the "accepted" one when we merge

```suggestion
          gh pr edit $PR_NUMBER --remove-label "status: in discussion" --repo $REPO
          gh pr edit $PR_NUMBER --add-label "status: accepted" --repo $REPO
```

---

Looking at the [flow](https://github.com/NixOS/rfcs/?tab=readme-ov-file#process-from-creation-to-merge) as well as  [this PR](https://github.com/NixOS/rfcs/pull/180), it seems like what we actually need to do is to remove the `status: FCP` label.

Are there cases where we jump straight from "in discussion" to "accepted"?

---

Ahh right, should be FCP